eagle-xl/registration.py

```diff
@@ -166,6 +166,70 @@
         ),
     )
 
+# Now neutral H masses and fractions
+
+try:
+    gas_mass = catalogue.masses.m_gas
+    H_frac = getattr(catalogue.element_mass_fractions, "element_0")
+    HI_frac = getattr(catalogue.species_fractions, "species_0")
+    H2_frac = getattr(catalogue.species_fractions, "species_2")
+
+    HI_mass = gas_mass * H_frac * HI_frac
+    H2_mass = gas_mass * H_frac * H2_frac
+    neutral_H_mass = HI_mass + H2_mass
+    neutral_H_mass.name = "$M_{HI + H_2}$"
+
+    setattr(self, "gas_neutral_H_mass_Msun", neutral_H_mass)
+
+    for aperture_size in aperture_sizes:
+        stellar_mass = getattr(catalogue.apertures, f"mass_star_{aperture_size}_kpc")
+        neutral_H_fraction = neutral_H_mass / stellar_mass
+        neutral_H_fraction.name = "$M_{HI + H_2} / M_*$" + f"({aperture_size} kpc)"
+
+        molecular_H_fraction = H2_mass / (H2_mass + stellar_mass)
+        molecular_H_fraction.name = (
+            "$M_{H_2} / (M_* + M_{H_2})$" + f"({aperture_size} kpc)"
```

Comment:
```suggestion
            f"$M_{{H_2}} / (M_* + M_{{H_2}})$ ({aperture_size} kpc)"
```

Reply:
The {} do not play well with the f hence that weird construction. And the change to M_HI above to match the style.